src/RazorSdk/Tasks/Microsoft.NET.Sdk.Razor.Tasks.csproj

```diff
@@ -42,7 +42,7 @@
     <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="Runtime" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" ExcludeAssets="Runtime" />
     <PackageReference Include="System.Reflection.Metadata" VersionOverride="$(SystemReflectionMetadataVersion)" Condition="'$(TargetFramework)'=='net472'" />
-    <PackageReference Include="System.Text.Json" Condition="'$(TargetFramework)'=='net472'" />
+    <PackageReference Include="System.Text.Json" VersionOverride="$(SystemTextJsonToolsetPackageVersion)" />
```

Comment:
In main, this reference is in a netframework conditioned itemgroup. Should it be here as well?

Reply:
I don't think it matters much since this should get dropped on core but might as well make the merge as easy as possible.